subiquity/models/subiquity.py

```diff
@@ -68,18 +74,30 @@ def __init__(self, common):
         if self.opts.dry_run:
             self.root = os.path.abspath(".subiquity")
             self.target = self.root
+
         self.locale = LocaleModel(common['signal'])
         self.keyboard = KeyboardModel(self.root)
         self.installpath = InstallpathModel(
             target=self.target,
             sources=common['opts'].sources)
         self.network = NetworkModel(support_wlan=False)
-        self.filesystem = FilesystemModel()
-        self.identity = IdentityModel()
         self.proxy = ProxyModel()
         self.mirror = MirrorModel()
-        self.snaplist = SnapListModel()
+        self.filesystem = FilesystemModel()
+
+        # Collect the models that produce data for the curtin config.
+        self._install_models = [
+            self.keyboard,
+            self.installpath,
+            self.network,
+            self.proxy,
+            self.mirror,
+            self.filesystem,
+            ]
```

Comment:
this list seems brittle, is it possible to just call render() on all models?  If an new or existing model adds curtin config, then this list needs updating.

Reply:
I sort of agree but there's no simple way to get a list of "all models" either really?

Reply:
One could interate the attributes of the object to see which ones are models.... but at the moment we'd need to like do a fuzzy match that `__class__.endswith('Model')` as all of them are just "objects" and don't inherit from anything. Maybe they should inherit all from a base-class Model, that at least implements `def render(): pass` but meh.